pom.xml

```diff
@@ -32,7 +32,7 @@
     <!-- Please note that when updating this dependency -->
     <!-- (i.e. due to a security vulnerability or bug) that the -->
     <!-- corresponding server dependency also needs updating.-->
-    <reactor-bom.version>Dysprosium-SR20</reactor-bom.version>
+    <reactor-bom.version>Dysprosium-SR21</reactor-bom.version>
```

Comment:
same question as https://github.com/neo4j/neo4j-java-driver/pull/962#pullrequestreview-704914900

Reply:
Just a typical dependency update. The server side will be updated too.